dataproc/create_cluster/create_cluster.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This sample shows how you can use the Cloud Dataproc Client library to create a
```

Comment:
```suggestion
// The dataproc package shows how you can use the Cloud Dataproc Client library.
```

Also, no newline after the comment. https://github.com/golang/go/wiki/CodeReviewComments#package-comments

Reply:
After changing this, the linter suggested `// Package dataproc shows how...`, what do you think?